freqtrade/freqtradebot.py

```diff
@@ -307,13 +314,17 @@ def get_target_bid(self, pair: str, ticker: Dict[str, float]) -> float:
 
         return used_rate
 
-    def _get_trade_stake_amount(self) -> Optional[float]:
+    def _get_trade_stake_amount(self, pair="") -> Optional[float]:
```

Comment:
i wouldn't add a default-value here - it adds no value (and this function is only called in 1 place outside of tests) - so calling it without a pair is next to impossible.